test/T42.stl.names.ref

```diff
@@ -2,19 +2,11 @@
 #include <functional> // std::binary_function
 #include <functional> // std::equal_to
 #include <functional> // std::less
-#include <iterator> // __gnu_cxx::__normal_iterator
-#include <map> // std::_Rb_tree_color
-#include <map> // std::_Rb_tree_const_iterator
+#include <list> // std::_List_node
```

Comment:
Most changes to this file is OS/libc++ dependent so results here is different from what i see on CentOS (our reference platform). Could you please revert all changes to this file except the removal of `pybind11::handle cl_type = cl;` and following new line? (if that too much trouble then revert the whole file and i will commit reference result from CentOS run later)